installer/Migration/V5_1_0/lang-string/general.yaml

```diff
@@ -110,3 +110,6 @@ langStrings:
   -
     value: 'Sat'
     unitId: sat
+  -
+    value: 'Accepts .docx, .doc, .odt, .pdf, .rtf, .txt up to 1MB'
+    unitId: accept_custom_format_file
```

Comment:
better to move under recruitment group. and rename as `allowed_resume_file_types`